modules/ROOT/pages/tutorial-user-systemd-unit-on-boot.adoc

```diff
@@ -1,18 +1,84 @@
-There are times when it's helpful to launch a user-level https://www.freedesktop.org/software/systemd/man/systemd.unit.html[systemd unit] without having to log in. For example, you may wish to launch a container that provides a network service or run an HPC job. Below is an example .fcc file that can be converted to ignition for the purpose of launching a systemd service for the user "sleeper". 
+= Launching a user-level systemd unit on boot
+
+NOTE: Make sure that you have completed the steps described in the xref:tutorial-setup.adoc[initial setup page] before starting this tutorial.
+
+In this tutorial, we will setup a user level systemd unit for an unprivileged user. There are times when it's helpful to launch a user-level https://www.freedesktop.org/software/systemd/man/systemd.unit.html[systemd unit] without having to log in. For example, you may wish to launch a container that provides a network service or run an HPC job. For this setup, we will add the following to a Fedora CoreOS config:
+
+* A user level systemd unit: `/home/sleeper/.config/systemd/user/linger-example.service`.
+* Enable it as a user level systemd service.
+
+== Setting up the systemd unit
+
+In this example, we will launch a systemd service for the user "sleeper". First, we add the systemd unit:
```

Comment:
This artificial split is missing a pre-requirement. The `sleeper` user does not exist in the system and has not been introduced up to this point.

I think you'd need to introduce a small `passwd` section for that before this step. It would be good to mention that `/home/sleeper` creation happens there too.